source/draw/RasterRgba.ooc

```diff
@@ -153,6 +153,8 @@ RasterRgba: class extends RasterPacked {
 		x, y, imageComponents: Int
 		requiredComponents := 4
 		data := StbImage load(filename, x&, y&, imageComponents&, requiredComponents)
+		if (data == null)
+			Exception new("Failed to load image: " + filename) throw()
```

Comment:
We could also throw directly from `StbImage load` and avoid some duplicated code.